docs/pages/admin-guides/access-controls/guides/hardware-key-support.mdx

```diff
@@ -224,6 +224,66 @@ on the policies of the key in the PIV slot.
 Make sure that the touch and PIN policies of the key satisfy the hardware key requirement
 for your cluster and roles.
 
+## Hardware Key Agent - Teleport Connect
+
+Teleport Connect provides the best UX for hardware key support with its UI elements for
```

Comment:
I would rephrase this paragraph to be less like an announcement, since it will be in the docs for long after the introduction of the feature. Maybe something like:

```markdown
Teleport Connect can act as a Hardware Key Agent, 
making hardware key requests on behalf of Teleport 
clients.
```